src/prepack-cli.js

```diff
@@ -170,7 +170,7 @@ function run(
       maxStackDepth: maxStackDepth,
       timeout: timeout,
       additionalFunctions: additionalFunctions,
-      enableDebugger: false, //always turn off debugger for now
+      enableDebugger: false, //always turn off debugger until debugger is fully built
```

Comment:
And then the debugger is always enabled by default? Why?

Reply:
This flag is hiding the debugger while it is being built. After it is built, whether it is used will be determined by the debuggerIsAttached check in the DebugChannel. This flag will be removed then.